### ISSUE TITLE
"jekyll trace" command doesn't give desired result  

### ISSUE THREAD

**Issue by zorbathegreek** [mmistakes/minimal-mistakes#1415]
This s not an issue as such, just a hint regarding the issue template.

The proposed command to retrieve info about an issue is
bundle exec jekyll --trace --verbose 
but I was getting error messages 
The command I found out to be giving the desired result is  
bundle exec jekyll serve --trace

Regards,

Anthony

**Comment by mmistakes** [comment]
Closing as this has nothing to do with the theme. Please file an issue upstream with `jekyll/jekyll` if you're having trouble with their commands.

**Comment by zorbathegreek** [comment]
Good idea; didn't know where to file this!

**Comment by zorbathegreek** [comment]
FYI
done
https://github.com/jekyll/jekyll/issues/6647

**Comment by mmistakes** [comment]
You're missing either `build` or `serve` that's why you're getting an error.

You're essentially just running `jekyll` without telling it to build your site or spin up a development server.

Try `bundle exec jekyll build --trace --verbose` instead.

**Comment by pathawks** [comment]
The issue that is being raised here is a problem with the comment at line 53 of the Minimal Mistakes issue template.

It is here that users are instructed to run `bundle exec jekyll —trace —verbose`

The original poster knows this command is incorrect; he is reporting the issue so that the issue template may be fixed.

**Comment by zorbathegreek** [comment]
Michael, they are throwing the ball back at me! And at you!
https://github.com/jekyll/jekyll/issues/6647#issuecomment-353524070
Sorry for complicating things but the Jekyll people say they can't fix it and that it were up to you. 
And sorry, I think I made a minimal mistake (haha), it's actually the below command that worked for me
bundle exec jekyll serve --trace --verbose 
so the "serve" is missing in your template (and the --verbose was missing in what I wrote you earlier).
Hope this helps, and... you're doing a great job!
